packages/create-vite/src/index.ts

```diff
@@ -428,6 +428,22 @@ async function init() {
 
   pkg.name = packageName || getProjectName()
 
+  if (pkgManager === 'bun') {
+    // run `vite` CLI with Bun runtime
+    // the --bun flag is required to override the node shebang
+    // after Bun 1.0 this will no longer be necessary
+    if (pkg.scripts) {
+      pkg.scripts = {
+        ...pkg.scripts,
+        dev: pkg.scripts.dev?.replace('vite', 'bunx --bun vite'),
```

Comment:
Can you use mutations so that the order is preserved?

Reply:
Done, good call.